src/main/java/App.java

```diff
@@ -3,5 +3,6 @@
 public class App {
```

Comment:
you have completely wrong structure. all files of your program should be in src/main/java package. but check where you created primitives package. fix this